ui/core/src/model/time.ts

```diff
@@ -26,6 +26,14 @@ export interface RelativeTimeRange {
   pastDuration: DurationString;
 }
 
+export interface TimeOption {
+  from: string;
+  to: string;
+  display: string;
+}
+
+export type DateTimeFormat = number | string;
+
```

Comment:
Feels like this is specific to the UI component and maybe should be part of its public API?